hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -194,12 +190,15 @@ public class ZookeeperFederationStateStore implements FederationStateStore {
   private int maxAppsInStateStore;
 
   /** Directory to store the delegation token data. **/
+  private Map<Integer, String> routerAppRootHierarchies;
```

Comment:
This PR must be missing a bunch no?

Reply:
I will finish this pr as soon as possible.